tests/ga/extension_emulator.py

```diff
@@ -225,30 +230,23 @@ def _extend_func(func):
         def wrapped_func(cmd, *args, **kwargs):
             return_value = func(cmd, *args, **kwargs)
 
-            config_dir = os.path.join(os.path.dirname(cmd), "config")
-            
-            regex = r'{directory}{sep}(?P<seq>{sequence})\.settings'.format(
-                directory=config_dir, sep=os.path.sep, sequence=r'[0-9]+'
-            )
-
-            seq = 0
-            for config_file in map(lambda filename: os.path.join(config_dir, filename), os.listdir(config_dir)):
-                if not os.path.isfile(config_file):
-                    continue
-
-                match = re.match(regex, config_file)
-                if not match:
-                    continue
-
-                if seq < int(match.group("seq")):
-                    seq = int(match.group("seq"))
+            prefix = kwargs['env'][ExtCommandEnvVariable.ExtensionSeqNumber]
```

Comment:
Always use the sequence number from Environment variables rather than fetching it from filesystem